packages/components/Molecules/Form/index.vue

```diff
@@ -84,6 +84,20 @@ export default {
       }
       return []
     }
+    // Validate if multiple select is a required field, then atleast one value should be selected.
+    VueFormGenerator.validators.rplSelectMultipleRequired = function (value, field) {
+      if (value && value.length >= field.min) {
+        return []
+      }
+      return ['Add a selection']
+    }
+    // Validate if multiple select has a max limiit of selection.
+    VueFormGenerator.validators.rplSelectMaxLimit = function (value, field) {
+      if (value && value.length <= field.max) {
+        return []
+      }
+      return ['More than ' + field.max + ' selections are not allowed']
+    }
```

Comment:
@MdNadimHossain I was planning to move these custom validators out of this component and into a separated js file. To make code more easy to maintain. Happy for you to do a little refactoring if you have time. But not necessary. Thanks!